fern/apis/server/definition/ingestion.yml

```diff
@@ -312,3 +316,17 @@ types:
     properties:
       successes: list<IngestionSuccess>
       errors: list<IngestionError>
+
+  OpenAIUsageSchema:
+    properties:
+      prompt_tokens: integer
+      completion_tokens: integer
+      total_tokens: integer
+      prompt_tokens_details: optional<map<string, integer>>
+      completion_tokens_details: optional<map<string, integer>>
+
+  UsageDetails:
+    discriminated: false
+    union:
+      - map<string, integer>
+      - OpenAIUsageSchema
```

Comment:
Why do we need this exactly?

Reply:
Currently we try to map within the SDK to convert from Openai to Langfuse schemas. Do you want to handle all of this server side now?

Reply:
Yes, I moved this to server side to also support users ingesting via API and to not duplicate logic in SDKs.